apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -150,18 +152,16 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
   metadataByAssetId: {},
 
   endAuction: async auctionId => {
-    set(state => {
-      state.dutchAuction.txInProgress = true;
-    });
+    const req = new TransactionPlannerRequest({ dutchAuctionEndActions: [{ auctionId }] });
+    await planBuildBroadcast('dutchAuctionEnd', req);
+    void get().dutchAuction.loadAuctionInfos();
+  },
 
-    try {
-      const req = new TransactionPlannerRequest({ dutchAuctionEndActions: [{ auctionId }] });
-      await planBuildBroadcast('dutchAuctionEnd', req);
-      void get().dutchAuction.loadAuctionInfos();
-    } finally {
-      set(state => {
-        state.dutchAuction.txInProgress = false;
-      });
-    }
```

Comment:
Realized we don't need to set `txInProgress`, since that's really just for controlling the submit button of the auction form. No need to disable it while an end/withdraw transaction is in progress.